modules/file-system/pre-existing-network-storage/README.md

```diff
@@ -45,6 +45,21 @@ were directories. This is important when working with files that were created by
 another source, but there may have performance impacts. The `_netdev` mount option
 denotes that the storage device requires network access.
 
+The following is an example of using `pre-existing-network-storage` with the `lustre`
+filesystem:
+
+```yaml
+- id: lustrefs
+  source: modules/file-system/pre-existing-network-storage
+  settings:
+    fs_type: lustre
+    server_ip: 192.168.227.11@tcp
+    local_mount: /data
+    remote_mount: /lustre/mgt
+```
+
+Note the use of the MGS NID (Network ID) in the `server_ip` field.
```

Comment:
nit: I assume this is referring to the `@tcp` (I am just unfamiliar with the `MSG NID` terminology). Maybe just go one step farther to make this clear:

_... in the `server_ip` field, in this case `@tcp`._